ERCS/eip-draft_open_ip_protocol.md

```diff
@@ -0,0 +1,179 @@
+---
```

Comment:
```suggestion
---
eip: 7548
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
done